universe/node/api_profile.go

```diff
@@ -72,10 +69,9 @@ func (n *Node) apiProfileUpdate(c *gin.Context) {
 	userProfile.OnBoarded = utils.GetPTR(true)
 
 	if err := n.db.UsersUpdateUserProfile(c, userID, userProfile); err != nil {
-		n.log.Error(errors.WithMessage(err, "failed to update user profile"))
-		c.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{
-			"message": "failed to update user profile",
-		})
+		err = errors.WithMessage(err, "failed to update user profile")
+		n.log.Error(err)
+		api.AbortRequest(c, http.StatusInternalServerError, "failed_to_update_user_profile", err)
```

Comment:
the same

Reply:
In this case `n.log.Error` not `n.log.Debug` as in others